apps/augury-backend/src/config/schemas/PortfolioGroup.ts

```diff
@@ -0,0 +1,25 @@
+import mongoose from 'mongoose';
+
+const schema = new mongoose.Schema({
+  name: {
+    type: String,
+    required: true,
+  },
+
+  color: {
+    type: String,
+    enum: ['white', 'red', 'green', 'blue', 'yellow'], // Enum definition
```

Comment:
same thing here

Reply:
Sorry this was merged before we addressed this. Is Addressed in #16